### DISCUSSION TITLE
Modifier grouping

### DISCUSSION THREAD

**Discussion by stephenhandley** [tailwindlabs/tailwindcss#12098] [Ideas]
It would be more readable if instead of doing
```
focus-visible:outline-none focus-visible:ring-2 focus-visible:ring-ring focus-visible:ring-offset-2
```

it was possible to just do
```
focus-visible:(outline-none ring-2 ring-ring ring-offset-2)
```

Assuming this has probably been requested before but didn't find existing issue on it.

**Comment by wongjn** [comment]
There have been suggestions about adding such a feature. The maintainers have addressed this request before, that you may wish to read up on if you have not already done so:
- [A Twitter thread from the creator of Tailwind, Adam Wathan](https://twitter.com/adamwathan/status/1461519820411789314)
- [A previous GitHub discussion thread with maintainer responses](https://github.com/tailwindlabs/tailwindcss/discussions/8337#discussioncomment-4032611)

**Reply by stephenhandley** [reply]
awesome thx!
